### PR TITLE
Brushup #95

### PR DESCRIPTION
**Pull request by tier940**
# Hide
* Combustion Generator
* Stirling Generator
* Qreative Powered Spawner
* Nutritious Stick
* Ender Pearl Powder
* Glowstone Dust(EIO)
* Ender Fragment
* Withering Dust
* Remote Awareness Upgrade
* Glowstone Nano-Particles
* Owl Egg
* EIO TNTs

# Add
* Wired Charger
* Energy Conduit
* Enhaned Energy Conduit
* Ender Energy Conduit
* Crystalline Alloy Endergy Conduit
* Crystalline Pink Slime Endergy Conduit
* Melodic Alloy Endergy Conduit
* Soularium Dust
* Crude Steel Dust
* Crystalline Alloy Dust
* Melodic Alloy Dust
* Stellar Alloy Dust
* Crystalline Pink Slime Dust

# Change
* Conductive Iron
	* 2700K -> 1700K
* Construction Alloy(Iron Alloy)
	* 5400K -> 1700K

# Update
* Experience Rod
* Coordinate Selector
* Pulsating Crystal
* Vibrant Crystal
* Ender Crystal
* Prescient Crystal
* Grains of Prescience
* Grains of Vibrant
* Grains of Piezallity
* Grains of the End
* Zombie Eletrode
* Z-Logic Controller
* Ender Resonator
* Skeletal Contractor
* Guardian Diode
* Tormented Enderman Head
* Totemic Capacitor

# Fix Formula
* Electrical Steel
* Energetic Alloy
* Vibrant Alloy
* Redstone Alloy
* Pulsating Iron
* Dark Steel
* Soularium
* End Steel
* Crystalline Alloy
* Melodic Alloy
* Stellar Alloy
* Crystalline Pink Slime
* Energetic Silver
* Vivid Alloy

### CODE REVIEW

**Review by Quarri6343 — CHANGES_REQUESTED** [review]
いくつか気がかりだったのでマークしておきました

File: overrides/scripts/materials.zs [hunk 1]
```diff
@@ -168,14 +170,18 @@ MaterialBuilder(32106, "dark_steel")
...
+    .color(0x5A4226)
+    .blastTemp(3600, "HIGH", 480, 600)
     .flags([
         "generate_plate"
     ])
-    .blastTemp(3600, "HIGH", 480, 600)
+    .components([
+        <material:gold> * 1,
+        <material:ash> * 1
+    ])
```
[the remark this answers is not in the document]
Comment (tier940) [reply overrides/scripts/materials.zs:182]: エラーするから故意にそうしてる

File: overrides/scripts/normal/EnderIO/02_Materials.zs [hunk 2]
```diff
@@ -371,56 +498,97 @@ JEI.hide(<enderio:item_material:32>);
...
-    .duration(100)
-    .EUt(480)
+    .duration(20)
+    .EUt(2)
+    .buildAndRegister();
+mixer.recipeBuilder()
+    .inputs([
+        <ore:itemVibrantPowder>,
+        <ore:dustEndstone>
+    ])
```
[the remark this answers is not in the document]
Comment (tier940) [reply overrides/scripts/normal/EnderIO/02_Materials.zs:587]: End Steel Dustだと鬼畜かなと思ったからEndstone Dustにしてた

File: overrides/scripts/normal/EnderIO/02_Materials.zs [hunk 3]
```diff
@@ -371,56 +498,97 @@ JEI.hide(<enderio:item_material:32>);
...
     .outputs([<enderio:item_material:34>])
-    .duration(100)
-    .EUt(480)
+    .duration(20)
+    .EUt(2)
+    .buildAndRegister();
+mixer.recipeBuilder()
+    .inputs([
+        <ore:itemVibrantPowder>,
+        <ore:dustEnderPearl>
```
[the remark this answers is not in the document]
Comment (tier940) [reply overrides/scripts/normal/EnderIO/02_Materials.zs:520]: 確かに。何か案ありますか...？

[the remark this answers is not in the document]
Comment (Quarri6343) [reply overrides/scripts/normal/EnderIO/02_Materials.zs:520]: shulker box dust? (shulker boxにレシピ追加)

**Review by Quarri6343 — CHANGES_REQUESTED** [review]

File: overrides/scripts/materials.zs (hunk 1 above)
[the remark this answers is not in the document]
Comment (tier940) [reply overrides/scripts/materials.zs:186]: 作成と分解では入れたのでクローズ

File: overrides/scripts/normal/EnderIO/02_Materials.zs (hunk 3 above)
[the remark this answers is not in the document]
Comment (tier940) [reply overrides/scripts/normal/EnderIO/02_Materials.zs]: 無理だったのでPlatinum Dustにしました

**Review by Quarri6343 — CHANGES_REQUESTED** [review]

**Review by Quarri6343 — APPROVED** [review]

**Pull request merged**